[16.0][MIG] account_mass_reconcile: Migrate to version 16.0

sonhd91 opened the pull request:
Port PR:
* https://github.com/OCA/account-reconcile/pull/428

mdietrichc2c committed:
Merge and migrate account_easy_reconcile and account_advanced_reconcile

oca-transbot committed:
OCA Transbot updated translations from Transifex

sbidoul committed:
[FIX] account_mass_reconcile: remove en.po that was erroneously created by transbot

florian-dacosta committed:
[FIX] account_mass_reconcile: Fix writeoff variables passed to reconcile method

pedrobaeza committed:
[MIG] account_mass_reconcile: Make modules uninstallable

pedrobaeza committed:
[MIG] account_mass_reconcile: Rename manifest files

mdietrichc2c committed:
[FIX] account_mass_reconcile: Remove now non-existing "Display partial" button

grindtildeath committed:
[MIG] account_mass_reconcile: Init mgration to 10.0

MiquelRForgeFlow committed:
[FIX] Readme display

Commit:
[MIG] account_mass_reconcile: Migration to odoo 11

MiquelRForgeFlow committed:
[FIX] Super call on static methods

MiquelRForgeFlow committed:
[FIX] Deprecated icon on some buttons

JordiBForgeFlow committed:
account_mass_reconcile: fix in python3 code conversion

osoul-dev committed:
Translated using Weblate (Arabic)

Currently translated at 78.2% (68 of 87 strings)

Translation: account-reconcile-11.0/account-reconcile-11.0-account_mass_reconcile
Translate-URL: https://translation.odoo-community.org/projects/account-reconcile-11-0/account-reconcile-11-0-account_mass_reconcile/ar/

MiquelRForgeFlow committed:
[ADD] account_mass_reconcile: assure  migration is ok

AdriaGForgeFlow committed:
[12.0][MIG] Migrate module account_mass_reconcile to v12

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-reconcile-12.0/account-reconcile-12.0-account_mass_reconcile
Translate-URL: https://translation.odoo-community.org/projects/account-reconcile-12-0/account-reconcile-12-0-account_mass_reconcile/

Commit:
[IMP] aditional depndency to trigger recompute properly

grindtildeath committed:
Revert "Merge pull request #276 from i-vyshnevska/12.0-account_mass_reconcile_transaction_ref"

This reverts commit 16832c31e1cef26cb052373ef83398d8662f4781, reversing
changes made to 7da56863b3528e49810542847f118d1a095a6d57.

AdriaGForgeFlow committed:
[IMP] account_mass_reconcile: black, isort

AdriaGForgeFlow committed:
[MIG] account_mass_reconcile: Migration to 13.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-reconcile-13.0/account-reconcile-13.0-account_mass_reconcile
Translate-URL: https://translation.odoo-community.org/projects/account-reconcile-13-0/account-reconcile-13-0-account_mass_reconcile/

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 54.8% (57 of 104 strings)

Translation: account-reconcile-13.0/account-reconcile-13.0-account_mass_reconcile
Translate-URL: https://translation.odoo-community.org/projects/account-reconcile-13-0/account-reconcile-13-0-account_mass_reconcile/nl/

florian-dacosta committed:
[IMP] account_mass_reconcile: black, isort, prettier

florian-dacosta committed:
[MIG] account_mass_reconcile: Migration to 14.0

florian-dacosta committed:
Manage write off, some refactore to manage better currency, partial reconciliation, and add some tests
Also add an advanced reconciliation method based on name. Could be very useful to actually match account move line based
on payment_reference field on invoices, as it is passed to account move line on name field

yvaucher committed:
account_mass_reconcile: Fix early commit by chunk

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (104 of 104 strings)

Translation: account-reconcile-14.0/account-reconcile-14.0-account_mass_reconcile
Translate-URL: https://translation.odoo-community.org/projects/account-reconcile-14-0/account-reconcile-14-0-account_mass_reconcile/es_AR/

kittiu committed:
[13.0][FIX] account_mass_reoncile, mass reoncile only posted entry

OCA-git-bot committed:
account_mass_reconcile 14.0.1.0.1

OCA-git-bot committed:
[IMP] update dotfiles [ci skip]

ofonseca-pyming committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (104 of 104 strings)

Translation: account-reconcile-14.0/account-reconcile-14.0-account_mass_reconcile
Translate-URL: https://translation.odoo-community.org/projects/account-reconcile-14-0/account-reconcile-14-0-account_mass_reconcile/es/

ofonseca-pyming committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (104 of 104 strings)

Translation: account-reconcile-14.0/account-reconcile-14.0-account_mass_reconcile
Translate-URL: https://translation.odoo-community.org/projects/account-reconcile-14-0/account-reconcile-14-0-account_mass_reconcile/es/

yvaucher committed:
[FIX] account_mass_reconcile: Fix reconciliation with empty name

The field name on account.move.line is not required anymore.
Avoid a call of `lower()` on None

MiquelRForgeFlow committed:
[FIX] account_mass_reconcile: reuse currency rate in test

LudLaf committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (104 of 104 strings)

Translation: account-reconcile-14.0/account-reconcile-14.0-account_mass_reconcile
Translate-URL: https://translation.odoo-community.org/projects/account-reconcile-14-0/account-reconcile-14-0-account_mass_reconcile/es/

OCA-git-bot committed:
account_mass_reconcile 14.0.1.0.2

OCA-git-bot committed:
account_mass_reconcile 14.0.1.1.0

Commit:
[IMP] account_mass_reconcile: black, isort, prettier

OCA-git-bot committed:
account_mass_reconcile 14.0.1.1.1

eugenichafer committed:
Translated using Weblate (Catalan)

Currently translated at 7.6% (8 of 104 strings)

Translation: account-reconcile-14.0/account-reconcile-14.0-account_mass_reconcile
Translate-URL: https://translation.odoo-community.org/projects/account-reconcile-14-0/account-reconcile-14-0-account_mass_reconcile/ca/

Commit:
[16.0][MIG] account_mass_reconcile: Migrate to version 16.0

dzungtran89 commented on account_mass_reconcile/tests/test_scenario_reconcile.py:
```diff
@@ -213,6 +213,9 @@ def test_reconcile_with_writeoff(self):
             }
         )
+        # mass_rec.flush_recordset()
+        # invoice.payment_id = payment.id
```
@sonhd91 ty, btw why do we need to comment this code?

sonhd91 replied:
Sorry my bad, I just update the branch, please check again

MiquelRForgeFlow committed:
[IMP] account_mass_reconcile: black, isort, prettier

MiquelRForgeFlow committed:
[MIG] account_mass_reconcile: Migration to 15.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-reconcile-15.0/account-reconcile-15.0-account_mass_reconcile
Translate-URL: https://translation.odoo-community.org/projects/account-reconcile-15-0/account-reconcile-15-0-account_mass_reconcile/
